enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -790,7 +790,32 @@ To answer this question, we need to understand the requirements around feature
 gates and understand whether just installing the gatewayclasses CRD may be
 problematic for some clusters.
 
-**Resolution**: TBD.
+**Resolution**: Yes, we should use a feature gate.  It is not just the installation of
+the CRDs that must be considered.  The additional controllers and watchers needed for
+managing Gateway API resources should be avoided if not needed. In addition, having an
+official feature gate is useful for setting expectations about whether a cluster may be
+upgraded. In Dev and Tech Preview, a cluster with this feature should not be upgradeable.
+Finally, if a user wants to install the Gateway API CRDs on their cluster for experimenting
+with an implementation other than OSSM/Istio, we should permit them to do this without
+interference from OpenShift Gateway API on OSSM.
+
+In Dev Preview we choose to use the `CustomFeatureNoUpgrade` variety of feature gate primarily
```

Comment:
Is it `CustomFeatureNoUpgrade` or `CustomNoUpgrade`? I thought `CustomeNoUpgrade` is in https://github.com/openshift/api/blob/master/config/v1/types_feature.go.

Reply:
Fixed.